packages/dev/core/src/Materials/Node/Blocks/PBR/reflectionBlock.ts

```diff
@@ -505,6 +516,7 @@ export class ReflectionBlock extends ReflectionTextureBaseBlock {
         this.forceIrradianceInFragment = serializationObject.forceIrradianceInFragment;
         if (this.texture) {
             this.texture.gammaSpace = serializationObject.gammaSpace;
+            this.useAlternateRoughRadiance = serializationObject.useAlternateRoughRadiance;

```

Comment:
```suggestion
            this.useAlternateRoughRadiance = !!serializationObject.useAlternateRoughRadiance;
```
to support code that would have `serializationObject.useAlternateRoughRadiance == undefined`.